expense/activities.go

```diff
@@ -57,8 +57,8 @@ func WaitForDecisionActivity(ctx context.Context, expenseID string) (string, err
 		logger.Info("waitForDecisionActivity failed to register callback.", "Error", err)
 		return "", err
 	}
-	body, err := ioutil.ReadAll(resp.Body)
-	_ = resp.Body.Close()
+	defer resp.Body.Close() // nolint:errcheck
+	body, err := io.ReadAll(resp.Body)
```

Comment:
I think we should leave this how it was and just change `ioutil` to `os`. We want to explicitly show we ignore errors and there is no benefit to changing this to a defer here.